java/src/test/java/me/everything/inbloom/BloomFilterTest.java

```diff
@@ -9,6 +9,102 @@
  */
 public class BloomFilterTest extends TestCase {
 
+    public void testCreateFilterWithBadParameters()
+    {
```

Comment:
I understand that there's no convention for braces in the project, I think we should go with omitting the new line before opening braces.
I'll fix in the rest of the project.